cmd/oci-runtime-tool/generate.go

```diff
@@ -296,12 +290,11 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 		gidMaps = context.StringSlice("gidmappings")
 	}
 
+	// Add default user namespace.
 	if len(uidMaps) > 0 || len(gidMaps) > 0 {
-		needsNewUser = true
+		g.AddOrReplaceLinuxNamespace("user", "")
```

Comment:
Well, sorry, it will generate an invalid config, but the user asked us to generate it...